src/test/java/Homework17.java

```diff
@@ -0,0 +1,72 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+    public class Homework17 extends BaseTest {
+    @Test
+    public  void addSongToPlaylist() throws InterruptedException {
+
+        String newSongAddedNotificationText = "Added 1 song into";
+
+        navigateToPage();
+        provideEmail("krista_ua86@gmail.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        searchSong("Ketsa");
+        clickViewAllBtn();
+        selectFirstSongResult();
+        clickAddToBtn();
+        choosePlaylist();
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226440053-ab36b999-aa31-4219-906e-15dd8623443f.png)

HINT


![image](https://user-images.githubusercontent.com/104761482/226443314-81dc3045-4368-435d-8ff3-be4c31e217e2.png)


![image](https://user-images.githubusercontent.com/104761482/226440278-f4194ee8-5076-4c13-b5d4-28295f53012f.png)